server/channels/store/sqlstore/user_store.go

```diff
@@ -1571,6 +1575,43 @@ func (us SqlUserStore) Search(rctx request.CTX, teamId string, term string, opti
 	return us.performSearch(query, term, options)
 }
 
+func (us SqlUserStore) SearchCommonContentFlaggingReviewers(term string) ([]*model.User, error) {
```

Comment:
Note to reviewer - thee two functions `SearchCommonContentFlaggingReviewers` and `SearchTeamContentFlaggingReviewers` are implemented in user store instead of content flagging SQLs tore so I can reuse the `SqlUserStore.UserSearchOptions` method.